TASVideos/Pages/Forum/Index.cshtml

```diff
@@ -15,18 +15,25 @@
 	</div>
 </fullrow>
 <row>
-	<div class="col-md-6">
+	<div class="d-none d-md-block col-md-6">
 		@* ReSharper disable once UnusedVariable *@
 		@foreach (var cat in Model.Categories.OrderBy(c => c.Ordinal).FirstHalf())
 		{
-		<partial name="_Category" model="cat" />
+			<partial name="_Category" model="cat" />
 		}
 	</div>
-	<div class="col-md-6">
+	<div class="d-none d-md-block col-md-6">
 		@* ReSharper disable once UnusedVariable *@
 		@foreach (var cat in Model.Categories.OrderBy(c => c.Ordinal).SecondHalf())
 		{
-		<partial name="_Category" model="cat" />
+			<partial name="_Category" model="cat" />
+		}
+	</div>
+	<div class="d-md-none">
+		@* ReSharper disable once UnusedVariable *@
+		@foreach (var cat in Model.Categories.OrderBy(c=>(c.Ordinal-7)%27)) // please don't ban me
```

Comment:
Definitely a bannable offense
But even more than that is the lack of spaces, please fix c => (c.Ordinal - 7) % 27)

If the order of forums had changed in the past 17 years I'd say this is a god awful solution.  But they haven't and won't change